### ISSUE TITLE
[Accessibility : Keyboard & navigation] Add bond page not accessible through keyboard

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/ginger#283]
Action dropdown is opened using keyboard.
Add Bond page is not opening through keyboard enter button click.

![add bond](https://cloud.githubusercontent.com/assets/13199397/14106841/8f0f4e78-f5d1-11e5-849c-816c8f6f7916.jpg)
